msg/OpenDroneIdSystem.msg

```diff
@@ -0,0 +1,15 @@
+uint64 timestamp
```

Comment:
Shouldn't this be uint32_t according to OPEN_DRONE_ID_SYSTEM? I think 64 bit is going to be costly

Reply:
Seems to be needed for uORB compatibility. Gives me a build error if timestamp is not uint64. 
`[ERROR] uORB topic files generator:
	generate_output_from_file:	'timestamp' field in OpenDroneIdSystem msg definition is not of type uint64 but rather of type uint32!
`